test/unit-test.js

```diff
@@ -0,0 +1,46 @@
+var expect = require('chai').expect;
+var assert = require('chai').assert;
+var should = require('chai').should;
+
+const fs = require('fs');
+const h5 = require('../index')
+const chai = require('chai')
+const chaiAsPromised = require('chai-as-promised')
+const TEST_FILE = fs.readFileSync('/Users/bcg/Desktop/Tracks/Salute.mp3');
+const TEST_DATAURL = `data:audio/mp3;base64,${TEST_FILE.toString('base64')}`;
+chai.use(chaiAsPromised)
+
+describe('test bundle', () => {
+  it('should work', () => {
+    assert.equal();
+  });
+});
+
+describe('file conversion process', () => {
+
+  it('should get the song\'s duration', (done) => {
+    return assert.eventually.equal(Promise.resolve(h5.getSongDuration('/Users/bcg/Desktop/Tracks/Salute.mp3')), 168.098, "This should return time in seconds").notify(done);
```

Comment:
Yea, this test will fail on my computer because I don't have a `bcg` user account and I don't have that file. Bummer. This happens a number of other times in this test suite, but this is the last time I'm mentioning it.